pyro/distributions/transforms/__init__.py

```diff
@@ -0,0 +1,22 @@
+from .batch_norm import BatchNormTransform
```

Comment:
nit: Would suggest adding `from __future__ import absolute_import, division, print_function` at the top of file and use absolute imports below `from transforms.householder import HouseholderFlow` for consistency with the rest of the codebase.

Reply:
Done!